extensions/eda/plugins/event_source/webhook.py

```diff
@@ -29,7 +31,11 @@
 @routes.post(r"/{endpoint:.*}")
 async def webhook(request: web.Request) -> web.Response:
     """Return response to webhook request."""
-    payload = await request.json()
+    try:
+        payload = await request.json()
+    except json.JSONDecodeError as e:
+        logger.warning("Wrong body request: failed to decode JSON payload: %s", e)
+        raise web.HTTPBadRequest(text="Invalid JSON payload") from None
```

Comment:
Why do we need raise exception from `None`?

Reply:
Hi @hsong-rh This is to pass ruff checks. When you raise an different exception in a try block, ruff enforce to use the "raise from" to have the whole trace. In this case we don't want it because there is no any info to provide for debugging, so it must be "None"

Ref: https://beta.ruff.rs/docs/rules/raise-without-from-inside-except/